mdit_py_plugins/admon/index.py

```diff
@@ -130,7 +130,10 @@ def admonition(state: StateBlock, startLine: int, endLine: int, silent: bool) ->
     token = state.push("admonition_open", "div", 1)
     token.markup = markup
     token.block = True
-    token.attrs = {"class": " ".join(["admonition", tag, *_extra_classes(markup)])}
+    token.attrs = {
+        "class": " ".join(["admonition", tag, *_extra_classes(markup)]),
+        "role": tag,
```

Comment:
`role` is for ARIA role. We should not propagate typical values such as `warning` there.

Reply:
… though it would make sense to set the `note` role for `!!! note`.

Reply:
Was just about to ask. So should all be `"note"` then (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/note_role)? I couldn't find how `role` was added in https://github.com/Python-Markdown/markdown for the example you provided in #93

Reply:
Sorry the role was an error on my part. I have some filter which adds some things on top of the raw markdown output :)

Reply:
No problem, I reverted the commits, so this should be ready with the fix for the quotes